x-pack/plugins/ml/server/lib/register_settings.ts

```diff
@@ -30,5 +34,41 @@ export function registerKibanaSettings(coreSetup: CoreSetup) {
         }),
       },
     },
+    [ANOMALY_DETECTION_ENABLE_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.enableAnomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Enable default time range for anomaly detection jobs.',
+      }),
+      value: false,
+      schema: schema.boolean(),
+      description: i18n.translate(
+        'xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeDesc',
+        {
+          defaultMessage: 'Use a default time range to view anomaly detection jobs.',
+        }
+      ),
+      category: ['Machine Learning'],
+    },
+    [ANOMALY_DETECTION_DEFAULT_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Time filter defaults',
+      }),
+      type: 'json',
+      value: `{
+  "from": "now-15m",
+  "to": "now"
+}`,
+      description: i18n.translate(
+        'xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeDesc',
+        {
+          defaultMessage: 'The default time range to view anomaly detection jobs.',
```

Comment:
> The default time range...

If you want a link to the list of accepted formats, perhaps it can be copied from the way data.advancedSettings.timepicker.quickRangesText uses {acceptedFormatsLink} in ui_settings.ts.  I'm not convinced that link is necessary here, however.